src/app/profile/settings/connected-accounts/connected-accounts.component.ts

```diff
@@ -26,30 +25,32 @@ export class ConnectedAccountsComponent implements OnDestroy, OnInit {
   contextUserName: string;
   cluster: string;
 
-  constructor(private contexts: Contexts,
-    private auth: AuthenticationService,
-    private userService: UserService,
-    private providerService: ProviderService) {
-    this.subscriptions.push(auth.gitHubToken.subscribe(token => {
+  private readonly subscriptions: Subscription[] = [];
+
+  constructor(
+    contexts: Contexts,
```

Comment:
out of curiosity, why the change away from private?

Reply:
The `private` keyword there makes the parameters become private members of the class, but these parameters were only referenced directly within the constructor body and nowehere else in the class, so there is no need to make them class members.